be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -177,23 +180,54 @@ class LakeTabletsChannel : public TabletsChannel {
     bool _is_incremental_channel{false};
 
     std::set<int64_t> _immutable_partition_ids;
+
+    // Profile counters
+    // Number of tablets
+    RuntimeProfile::Counter* _tablets_num = nullptr;
+    // Number of times that open() is called
+    RuntimeProfile::Counter* _open_counter = nullptr;
+    // Accumulated time of open()
+    RuntimeProfile::Counter* _open_timer = nullptr;
+    // Number of times that add_chunk() is called
+    RuntimeProfile::Counter* _add_chunk_counter = nullptr;
+    // Accumulated time of add_chunk()
+    RuntimeProfile::Counter* _add_chunk_timer = nullptr;
+    // Number of rows added to this channel
+    RuntimeProfile::Counter* _add_row_num = nullptr;
+    // Accumulated time to wait for memtable flush in add_chunk()
+    RuntimeProfile::Counter* _wait_flush_timer = nullptr;
+    // Accumulated time to wait for async delta writers in add_chunk()
+    RuntimeProfile::Counter* _wait_writer_timer = nullptr;
```

Comment:
Is wait_writer_timer a typo? suggest to use wait_write_timer. the names seem puzzling.
can we use other names, for examples
1. memtable_filling_time;
2. full_memtable_wait_time;
3. segment_build_time;
4. segment_flush_time;

Reply:
use `wait_write_timer` instead